### DISCUSSION TITLE
GraphQL API errors out when fetching members after the first few pages

### DISCUSSION THREAD

**Discussion by fasterthanlime** [community/community#101402] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

This one is easy to reproduce if you have access to an account with 100+ sponsors (I'm hoping GitHub employees can masquerade as any account).

Here's my GraphQL query:

```graphql
query($first: Int, $after: String) {
  viewer {
    sponsors(first: $first, after: $after) {
      pageInfo {
        endCursor
      }
      nodes {
        ... on User {
          login
          name
          sponsorshipForViewerAsSponsorable {
            privacyLevel
            tier {
              monthlyPriceInDollars
              isOneTime
            }
          }
        }
        ... on Organization {
          login
          name
          sponsorshipForViewerAsSponsorable {
            privacyLevel
            tier {
              monthlyPriceInDollars
              isOneTime
            }
          }
        }
      }
    }
  }
}
```

Running it on the GraphQL explorer with variables set to:

```json
{
   "first": 100
}
```

...works at first, but eventually, it errors out, for example with these variables (for my account):

```json
{
  "first": 100,
  "after": "Y3Vyc29yOnYyOpIAzgGMAfI="
}
```

(It says to mention `B689:3E7F:5F09547:C642454:65AE56D8`)

Note that it works with `"first": 4`, which I'm assuming is exactly how many sponsors I have right now.

**Comment by cheshire137** [comment]
Hi, thanks for the report! I'm an engineer at GitHub working on Sponsors. I was able to find the underlying exception, thanks to you sharing the request ID. We'll look into this.

**Comment by cheshire137** [comment, marked as answer]
All right, we believe this is fixed. Would you give your query another try, see if the error reoccurs? Thanks!

**Reply by fasterthanlime** [reply]
I can verify I'm not getting a 500 anymore with this request. Thanks so much addressing this quickly! ✨
